apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/previewEditor/TemplateEditor.tsx

```diff
@@ -218,6 +223,24 @@ export const TemplateEditor: FunctionComponent<TemplateEditorProps> = ({
                 </GoAFormItem>
               </>
             </Tab>
+            <Tab
+              testId={`pdf-test-generator`}
+              label={<PdfEditorLabelWrapper>Template Variables</PdfEditorLabelWrapper>}
```

Comment:
it should be `Variable Assignments` i think as per the story https://goa-dio.atlassian.net/jira/software/projects/CS/boards/13?selectedIssue=CS-1904

Reply:
The design picture shows "Template Variables".

Reply:
![image](https://user-images.githubusercontent.com/31360331/215902398-afe4c71e-f107-42e8-8979-b9ca530d3064.png)
there is a note that mentions it